src/main/java/service/FileService.java

```diff
@@ -0,0 +1,16 @@
+package service;
+
+import java.io.IOException;
+import java.net.URISyntaxException;
+
+import utils.FileIoUtils;
+import webserver.entity.RequestHeader;
+import webserver.entity.ResponseEntity;
+
+public class FileService {
+    public ResponseEntity serveFile(final RequestHeader request) throws IOException, URISyntaxException {
+        String path = request.getPath();
+        byte[] staticFileData = FileIoUtils.loadFileFromClasspath(path);
+        return ResponseEntity.of(path, staticFileData);
+    }
```

Comment:
`FileService`는 `UserService`와 동일한 계층의 코드로 보이는데요. 
그렇다면 이 응답의 반환 값도 `HttpStatus`, `HttpHeader` 등을 포함한 `Http Response`이 아니라 body로 전달할 `byte[]` 값만 받아오는게 좋지 않을까 싶어요. 🤔

그런데 그런 관점에서보면 이 `FileService`의 `serveFile` 메서드 로직은 아래와 같이 될 것 같은데
굳이 서비스로 분류할 필요가 없어질 것 같네요. 🤔

```suggestion
    public ResponseEntity serveFile(final RequestHeader request) throws IOException, URISyntaxException {
        String path = request.getPath();
        return FileIoUtils.loadFileFromClasspath(path);
    }
```

Reply:
피드백 주신대로 `FileService`의 `serveFile`는 파일 데이터를 byte[]로 반환하도록 변경하고 있습니다.
이때 서비스로 분류하지 않을 이유가 어떤 부분인지 궁금합니다!

Reply:
수정 전에 FileService.`serveFile` 메서드의 반환 값은 Http 응답 즉, `ResponseEntity` 객체였는데, 그에 반해서 UserService의 `createUser` 메서드의 반환 값은 없었어요.  (Http 응답 객체였다면 body가 없는 `ResponseEntity` 객체이지 않았을까요? 🙂)

그래서 두 `Service`는 동일한 계층이기 때문에 통일된 결과가 필요하다고 생각했어요. 😉 (둘 다, Http 응답 객체이거나 아니거나)

그런데 File Service는 아래와 같이 Controller에서 파일을 읽어오는데 쓰이고있는데요. 
<img width="658" alt="image" src="https://github.com/next-step/jwp-was-kakao/assets/47850258/bf32307f-c32c-4b08-9bad-2cbb81f429f3">

이 코드를 아래와 같이 별도의 서비스로 뽑아내지 않고도 충분히 가독성도 좋고 또한 FileService.`serveFile`가 이 로직 외에서 사용되는 위치도 없다보니 별도의 서비스 로직으로 추출한 이점을 딱히 못느끼는 것 같아요. 🤔

```java
        try {
            byte[] file = FileIoUtils.loadFileFromClasspath(request.getPath());
            return ResponseEntity.of(request.getHeader().getPath(), file);
        } catch (Exception e) {
            throw new IllegalArgumentException("파일이 존재하지 않습니다.");
        }
```